src/index.tsx

```diff
@@ -384,6 +385,7 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
       adjustToContentHeight,
       alwaysOpen,
       closeAnimationConfig,
+      dragToss = 0.05,
```

Comment:
```suggestion
      dragToss,
```

You can remove the default value here, since you already specified it in the defaultProps